### PR TITLE
add piff and flags

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds piff and flags.

### CODE REVIEW

File: pizza_cutter/des_pizza_cutter/_des_info.py [hunk 1]
```diff
@@ -138,22 +123,28 @@ def get_des_y3_coadd_tile_info(*, tilename, band, campaign, medsconf):
...
-        ii['sky_bnds'] = sky_bnds
-        ii['ra_ccd'] = ra_ccd
-        ii['dec_ccd'] = dec_ccd
-        ii['ccd_bnds'] = Bounds(0, nrow-1, 0, ncol-1)
+        ii['psfex_psf'] = galsim.des.DES_PSFEx(ii['psf_path'])
+
+        # piff.  TODO make the piff_run configurable
+        piff_data = load_piff_from_image_path(
+            image_path=ii['image_path'],
+            piff_run='y3a1-v29',
```
Comment (beckermr) [inline pizza_cutter/des_pizza_cutter/_des_info.py]: I think we can pass the medsconf here? Or should we add another input?

File: pizza_cutter/des_pizza_cutter/_piff_tools.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        ok = False
+
+    if info['ccdnum'] == 31:
+        logger.info('skipping ccd 31 for exp %s' % expnum)
+        ok = False
+
+    return ok
+
+
+@lru_cache(maxsize=128)
```
Comment (beckermr) [inline pizza_cutter/des_pizza_cutter/_piff_tools.py]: What is the memory footprint of each Piff object? We should make sure the cache is not too big.

File: pizza_cutter/des_pizza_cutter/_piff_tools.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+def _get_piff_psf(psf_path):
+    """
+    load a piff.PSF object from the specified file
+    """
+    logger.info('reading: %s' % psf_path)
+    return piff.read(psf_path)
+
+
+@lru_cache(maxsize=128)
```
Comment (beckermr) [inline pizza_cutter/des_pizza_cutter/_piff_tools.py]: Ditto here for the memory footprint.

File: pizza_cutter/des_pizza_cutter/_des_info.py (hunk 1 above)
Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_des_info.py]: I think a piff_run input is the right thing to do.

File: pizza_cutter/des_pizza_cutter/_piff_tools.py (hunk 2 above)
Comment (esheldon) [reply pizza_cutter/des_pizza_cutter/_piff_tools.py]: the piff files are 350k and the summary file is about 2M

Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_piff_tools.py]: Mmmmk. This is probably fine then.

File: pizza_cutter/des_pizza_cutter/_des_info.py (hunk 1 above)
Comment (esheldon) [reply pizza_cutter/des_pizza_cutter/_des_info.py]: agreed

Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_des_info.py]: mmk I will do this after this one is merged.

**Pull request merged**